### PR TITLE
Allows filter to detect GIR classes

### PR DESCRIPTION
**Pull request by w-nicole**
Changed regex to detect more Subjects with Prereq for GIR classes.

### CODE REVIEW

File: src/components/ClassInfo.vue [hunk 1]
```diff
@@ -263,7 +263,16 @@ export default {
     subjectsWithPrereq: function () {
       const currentID = this.currentSubject.subject_id;
       const currentDept = currentID.substring(0, currentID.indexOf('.'));
-      var IDMatcher = new RegExp('(^|[^\\da-zA-Z])' + currentID.replace(/[.*+?^${}()|[\]\\]/g, '\\$&') + '(?![\\da-zA-Z])');
+
+      var thisGIRAttr = this.currentSubject.gir_attribute;
+
+      if(thisGIRAttr == undefined){
+        var IDMatcher = new RegExp('(^|[^\\da-zA-Z])' + currentID.replace(/[.*+?^${}()|[\]\\]/g, '\\$&') + '(?![\\da-zA-Z])');
```
Comment (georgiashay) [inline src/components/ClassInfo.vue:270]: The IDMatcher variable is declared separately within the if and else blocks.  While this does work fine (Javascript will "hoist" the variable so it exists outside of the if statement), it is better practice/clearer to define this variable above the if statement (just "var IDMatcher;") and then assign to it within the if statement.

**Review by georgiashay — CHANGES_REQUESTED** [review]
Everything works great!  Just one stylistic/best practices comment but ready to merge after that is addressed!

**Comment by w-nicole** [comment]
I committed a new version of the code with the style changes. It's visible in Commits. Please let me know if there's anything else I should be fix, or if I didn't push the commit correctly. Thank you for all of your help!

**Comment by georgiashay** [comment]
Thanks for making the changes!  I think it would be clearer to assign thisGIRAttr on one line inside the else statement (missed this the first time).  Again a minor thing, let me know if you think it is better the original way.

**Comment by georgiashay** [comment]
Pushed lint to trigger Travis (just updated our testing so it doesn't stop working after the new year).  You'll need to pull these changes before working on it (git pull).  If you've already worked on it, commit your changes and then run git pull to get my updates.

**Comment by w-nicole** [comment]
I changed the variable assignment to be in one line for the else statement.

**Comment by georgiashay** [comment]
Everything looks good!  Merging

**Pull request merged**